--- supertuxkart-0.6.1a/data/Makefile.am~	2009-08-19 11:46:35.000000000 +0200
+++ supertuxkart-0.6.1a/data/Makefile.am	2009-08-19 12:01:16.000000000 +0200
@@ -10,7 +10,6 @@
  $(shell find $(srcdir) -name "*.cup") \
  $(shell find $(srcdir) -maxdepth 1 -name "*.challenge") \
  $(shell find $(srcdir) -name "*.collectable") \
- $(shell find $(srcdir) -name "*.projectile") \
  CREDITS
 
 desktopdir = $(prefix)/share/applications
