--- supertuxkart-0.6/data/supertuxkart_desktop.template~	2009-01-22 14:42:20.000000000 +0100
+++ supertuxkart-0.6/data/supertuxkart_desktop.template	2009-01-22 14:42:21.000000000 +0100
@@ -1,15 +1,16 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=SuperTuxKart
-Icon=@PREFIX@/share/pixmaps/supertuxkart_64.xpm
+Icon=supertuxkart_32.xpm
 GenericName=A kart racing game
-GenericName[de_DE]=Ein Kart-Rennspiel
-GenericName[fr_FR]=Un jeu de karting
+GenericName[de]=Ein Kart-Rennspiel
+GenericName[fr]=Un jeu de karting
 GenericName[gl]=Xogo de carreiras con karts
-GenericName[ro_RO]=Un joc de curse cu carturi
-Exec=@PREFIX@/games/supertuxkart --log=file
+GenericName[ro]=Un joc de curse cu carturi
+GenericName[pl]=Gra wyścigowa
+Exec=supertuxkart --log=file
 Terminal=false
 StartupNotify=false
 Type=Application
-TryExec=@PREFIX@/games/supertuxkart
 Categories=Game;ArcadeGame;
+# vi: encoding=utf-8
