--- supertuxkart-0.7alpha3/configure.ac~	2010-09-24 03:21:15.000000000 +0200
+++ supertuxkart-0.7alpha3/configure.ac	2010-09-29 13:23:49.776109670 +0200
@@ -404,7 +404,7 @@
 # =================
 # Check for gettext
 # =================
-CHECK_GETTEXT(external)
+#CHECK_GETTEXT(external)
 #disable/enable addons support
 AC_ARG_WITH(addons,[AS_HELP_STRING([--with-addons],
               [This option is used to enable/disable addons support])])
--- supertuxkart-0.7alpha3/src/utils/translation.cpp~	2010-09-24 01:27:23.000000000 +0200
+++ supertuxkart-0.7alpha3/src/utils/translation.cpp	2010-09-29 13:24:44.236109680 +0200
@@ -33,6 +33,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <iostream>
+#include <libintl.h>
 
 #include "irrlicht.h"
 
@@ -113,12 +114,7 @@
     #endif
 #endif
     
-#if ENABLE_NLS
     const char* original_t = gettext(original);
-#else
-    m_converted_string = core::stringw(original);
-    return m_converted_string.c_str();
-#endif
 
     /*
     std::cout << "--> original_t==original ? " << (original_t==original) << std::endl;
